src/mission_start.cpp

```diff
@@ -1719,3 +1719,34 @@ void mission_start::reveal_cathedral( mission *miss )
 {
     reveal_any_target( miss, { "cathedral_1", "museum" } );
 }
+
+void mission_start::reveal_refugee_center( mission *miss )
```

Comment:
I've mentioned that I'm willing to move the reveal stuffs to json, I have some WIP but it is by no way ready to commit. Hence, as for the previous PR with reveals, it is out of scope.